pkg/controllers/v1alpha1/dataload/dataload_controller.go

```diff
@@ -120,14 +120,14 @@ func (r *DataLoadReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	}
 	ctx.RuntimeType = boundedRuntime.Type
 
-	var runtime runtime.Object
+	var fluidRuntime runtime.Object
```

Comment:
May I know the reason of changing this?

Reply:
> May I know the reason of changing this?

the variable name runtime collide  with the import package name runtime

Reply:
Does it matter in compiling or runtime?

Reply:
Now it does not affect, I just want to use a more elegant naming. I can rollback it to the original name if you think so

Reply:
No problem. I'm just curious about the reason.